ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/DynamoDBKeyValue.java

```diff
@@ -153,12 +153,14 @@ public String readSingle(String key) throws Exception {
                 .withConsistentRead(config.consistentRead());
             item = table.getItem(spec);
             if (item == null) {
-                return null;
+            return null;
             }
         } catch (AmazonServiceException ase) {
             amazonServiceException(ase);
```

Comment:
NdBench plugin ideally should throw an exception when there an error processing operation, if the plugin swallows the exception there is no way for NdBench driver to know the failed operation, hence in the catch block, we should be re-throwing the exception rather than returning error message